pyipn/geometry.py

```diff
@@ -32,8 +32,8 @@ def get_seperation_angle(self, grb):
         """
 
         return self._skycoord.separation(grb.location.coord).rad
-
-
+    
+#why pointing and location class different
 class Location(object):
```

Comment:
The location is where the spacecraft is in space. The pointing is what direction in the sky the detector is facing.